### DISCUSSION TITLE
having the possibility to select an existing  picture from the gallery

### DISCUSSION THREAD

**Discussion by hubyhuby** [GetPublii/Publii#1392] [Ideas]
I am using often the same pictures for the hero image, also I have to redeowload each time the same picture.
My media folder is growing fast, but I have only 10 pitures for all my posts :D

**Comment by bobmitro** [comment]
Use the file manager.

**Reply by candidexmedia** [reply]
@bobmitro In your opinion, would it be more sound (from a speed/bandwidth perspective) to use a File Manager, _or_ use a dedicated CDN for media like [Cloudinary](https://cloudinary.com/) or even something like [Imgur](https://imgur.com/upload)?

**Comment by hubyhuby** [comment]
The result is duplicated content on images for google. It doesn t index the same image in different posts ;) That make sens.
I agree this is an edge case, but it s little improvement, at least for me.

List of NOT indexed images : 
![image](https://github.com/GetPublii/Publii/assets/26340942/1bd4b35a-0cc5-4108-a9c6-e88e7e6be7ac)

**Reply by bobmitro** [reply]
That seems obvious. That's why I suggested using a file manager. However, if you want to use the same image from different locations, it's worth taking care of having different alts and titles, though I'm not sure if Google will treat them as different images.

**Reply by Simon99de** [reply]
i also think that the file manager should be better integrated so that if you want to select an image, you can also select one in the file manager, as this makes it easier to set the links / links automatically. subfolders such as icons, adverts, ... could also be helpful to keep it organised if you have multiple images, it would also save memory.

how would the function affect the performance? better / worse or the same?

Edit:
Suggested improvement:
https://github.com/GetPublii/Publii/discussions/2152#discussion-8184820
